plugins/outputs/kafka/sample.conf

```
@@ -87,6 +87,10 @@
   ##   * now: Uses the time of write
   # producer_timestamp = metric
 
+  ## Metric name header
+  ## This option adds metric name as kafka header, with specified key:
+  # metric_name_header = "metric_name"
+
```

By convention the sample config _should_ specify the default value for commented settings, so how about
```suggestion
  ## Add metric name as specified kafka header if not empty
  # metric_name_header = ""

```